src/app/components/create-retro-form/create-retro-form.component.html

```diff
@@ -0,0 +1,18 @@
+<div md-dialog-content>
+    <form #createRetroForm="ngForm">
+        <p>
+            <md-input-container>
+                <input mdInput placeholder="What's your name?"/>
+            </md-input-container>
+        </p>
+        <p>
+            <md-input-container>
+                <input mdInput placeholder="Retro ID" [(ngModel)]="retroId" name="retroIdInput" required/>
```

Comment:
Might wanna rename this to "Retro Name" or "Retro Title" since it doesn't actually correspond to the retro ID (I think we decided to use the db-generated ID for that).